docs/upload-stable.sh

```diff
@@ -7,7 +7,7 @@ set -o nounset
 
 aws configure set preview.cloudfront true
 aws --profile mitmproxy \
-    s3 sync --acl public-read ./public s3://docs.mitmproxy.org/stable
+    s3 sync --delete --acl public-read ./public s3://docs.mitmproxy.org/stable
```

Comment:
Mental note for future us: DO NOT place "unmanaged" files into the buckets. Everything has to go through the process + hugo.